app/src/main/java/com/kodeco/android/coordplot/MainActivity.kt

```diff
@@ -24,6 +42,11 @@ class MainActivity : ComponentActivity() {
 
 @Composable
 fun PlotSurface() {
+//    var xPercent by rememberSaveable { mutableStateOf(0.5f) }
+//    var yPercent by rememberSaveable { mutableStateOf(0.5f) }
```

Comment:
nit: you can remove the commented code.  Good job practicing with both remember and rememberSaveable tho!

Reply:
Done